src/__tests__/RenderParentDashboard.test.js

```diff
@@ -12,13 +12,18 @@ afterEach(() => {
   cleanup();
 });
 
-jest.mock('@okta/okta-react', () => ({
-  useOktaAuth: () => {
+jest.mock('@auth0/auth0-react', () => ({
+  Auth0Provider: ({ children }) => children,
+  withAuthenticationRequired: (component, _) => component,
+  useAuth0: () => {
     return {
-      authState: {
-        isAuthenticated: true,
+      isAuthenticated: false,
```

Comment:
Also in this test, the is Authenticated boolean has changed. See comment in ParentDashboardContainer.test.js